### PR TITLE
Add support for python 3

### PR DESCRIPTION
**Pull request by wo0dyn**
## Related issues
- #6 
## ChangeLog
- Add Python3.4 to tox matrix
- Add Python versions to setup.py
- Fix issue with StringIO w/ python3
- Remove depreciated warning about assertEquals()
- Fix generator.next()
- Use format() method to write in query_log
## Screenshot

screenshot from 2016-06-23 13 47 53

:tada:

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27-django{16,17,18},flake8
+envlist = py27-django{16,17,18},py34-django{16,17,18},flake8
```
Comment (FlogFr) [inline tox.ini]: not possible via:

```
{py27,py34}-django{16,17,18},flake8
```

?

**Comment by FlogFr** [comment]
Thanks for the PR
LGTM

File: tox.ini (hunk 1 above)
Comment (wo0dyn) [inline tox.ini]: Ah maybe. I'll try that. :wink:

Comment (wo0dyn) [inline tox.ini]: You're right, it works!

``` console
$ tox --list
py27-django16
py27-django17
py27-django18
py34-django16
py34-django17
py34-django18
flake8
```

I amended my commit.

**Pull request merged**